src/main/java/com/microexam/classapi/dto/ClassDTO.java

```diff
@@ -0,0 +1,21 @@
+package com.microexam.classapi.dto;
+
+import lombok.*;
+import org.springframework.validation.annotation.Validated;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.sql.Timestamp;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Validated
+public class ClassDTO {
+    private int classId;
+    @NotBlank(message = "Class name is required!")
```

Comment:
This is not necessary?

Reply:
This is for @NotBlank annotation

Reply:
Why is it not necessary? It is necessary that we do validation in the backend too

Reply:
And @NotBlank checks for both an empty string and a null value